src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int count = 4;

```

Comment:
```suggestion
    private static final int COUNT = 4;
```
constants should be static and have names in UPPER_CASE